src/utils/export.ts

```diff
@@ -18,3 +20,93 @@ export const exportAsJSON = async (id: string, name?: string) => {
+  const authorsHidden = !board.showAuthors ? `- ${t("MarkdownExport.showAuthorsDisabled")}\n` : "";
+  return `${header + boardId + participantsNumber + authorsHidden}\n`;
+};
+
+const mdVotesPerNote = (noteId: string, votings: VotingType[]) => {
+  const votes = votings ? getNoteVotes(noteId, votings) : 0;
+  return votes ? `${votes} ${votes === 1 ? t("MarkdownExport.vote") : t("MarkdownExport.votes")}` : "";
+};
+
+const mdBrackets = (addBrackets: boolean, input: string) => (addBrackets ? `(${input})` : input);
+
+const mdNote = (note: NoteType, author: string, votes: string, stack: string) => {
+  const nested = stack === "" && note.position.stack;
+  const bracketsRequired = !!author || !!votes;
+  const separator = !!author && !!votes ? ", " : "";
+  return `${nested ? "\n    " : "\n"}- ${note.text} ${mdBrackets(bracketsRequired, `${author + separator + votes}`)}${stack}`;
+};
+
+const mdStack = (notesInStack: NoteType[], boardData: BoardDataType) =>
+  notesInStack.length > 0
+    ? `${notesInStack
+        .map((n) =>
+          mdNote(
+            n,
+            boardData.board.showAuthors ? getAuthorName(n.author, boardData.participants) : "",
+            boardData.board.showAuthors ? mdVotesPerNote(n.id, boardData.votings) : "",
+            ""
+          )
+        )
+        .join("")}`
+    : "";
+
+const mdNotesPerColumn = (columnId: string, boardData: BoardDataType) =>
+  boardData.notes
+    .filter((n) => !n.position.stack && n.position.column === columnId)
+    .sort((a, b) => compareNotes(a, b, boardData.votings))
+    .map((n) => {
+      const votes = boardData.board.showVoting ? mdVotesPerNote(n.id, boardData.votings) : "";
+      const author = boardData.board.showAuthors ? getAuthorName(n.author, boardData.participants) : "";
+      const stack = mdStack(getChildNotes(boardData.notes, boardData.votings, n.id), boardData);
+      return mdNote(n, author, votes, stack);
+    })
+    .join("");
+
+const mdColumns = (boardData: BoardDataType) => {
+  const header = `## ${t("MarkdownExport.columns")}\n`;
+  const columnList = boardData.columns
+    .filter((c) => boardData.notes.filter((n) => n.position.column === c.id).length > 0)
+    .map(
+      (c: ColumnType) => `### ${c.name} (${boardData.notes.filter((n) => n.position.column === c.id).length} ${t("MarkdownExport.notes")})\n${mdNotesPerColumn(c.id, boardData)}`
+    )
+    .join("\n\n");
+  return `${header + columnList}\n\n`;
+};
+
+const mdBranding = () =>
+  `> ${i18n.t("PrintView.GeneratedOn")} [scrumlr.io](https://www.scrumlr.io)  \n${t("PrintView.ProvidedBy")} [inovex](https://www.inovex.de)  \n\n![Scrumlr Logo](${
+    process.env.REACT_APP_PUBLIC_URL
+  }/scrumlr-logo-light.svg)`;
+
+const mdTemplate = (boardData: BoardDataType) =>
+  mdBoardHeader(boardData.board.name) + mdColumns(boardData) + mdBoardProperties(boardData.board, boardData.participants) + mdBranding();
```

Comment:
```suggestion
  mdBoardHeader(boardData.board.name || "scrumlr.io") + mdBoardProperties(boardData.board, boardData.participants) + mdColumns(boardData) + mdBranding();
```

Reply:
I used the existing constant `DEFAULT_BOARD_NAME` instead :)